### PR TITLE
Institute of Mathematical Science - journals creation

### PR DESCRIPTION
**Pull request by POBrien333**
to go with: https://github.com/citation-style-language/styles/pull/2824

### CODE REVIEW

**Comment by POBrien333** [comment]
does either of you use Latex? They got some style files for the various journals of the IMS here: http://www.e-publications.org/ims/support/

**Comment by rmzelle** [comment]
> does either of you use Latex?

I used it once in 2003-04 for my bachelor thesis, but found it rather time-consuming and distracting. So no.

File: ims/_journals.tab [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Title	Title-short	ISSN	Documentation
+The Annals of Applied Probability	AAP	1050-5164	http://www.imstat.org/aap/manprep.html
+The Annals of Probability	AOP	 	0091-1798	http://www.imstat.org/aop/manprep.htm
+The Annals of Statistics	AOS	 	0090-5364	http://www.imstat.org/aos/manprep.html
+Statistical Science	STS	x	http://www.imstat.org/sts/manprep.html
+The Electronic Journal of Statistics	EJS	1935-7524	http://www.imstat.org/ejs/mansub.html
```
Comment (rmzelle) [inline ims/_journals.tab]: @POBrien333, I assume you checked all journals? You listed more than 5 at https://github.com/citation-style-language/styles/pull/2824#issue-243386318.

**Comment by adam3smith** [comment]
I use LaTeX enough to generate a couple of sample bibliographies, if that's the question?

**Comment by POBrien333** [comment]
@adam3smith

That would be helpful, if you could do that.

File: ims/_journals.tab (hunk 1 above)
Comment (POBrien333) [inline ims/_journals.tab]: That's the ones I checked and think that they should follow the style I made. With those LaTeX citationstyles (or whatever they are) I am unsure now though and would like to check this.

**Comment by rmzelle** [comment]
> I use LaTeX enough to generate a couple of sample bibliographies, if that's the question?

@adam3smith, bump :).

**Comment by adam3smith** [comment]
[acmtrans-ims.pdf](https://github.com/citation-style-language/journals/files/1184190/acmtrans-ims.pdf)
[Imsart-number.pdf](https://github.com/citation-style-language/journals/files/1184191/Imsart-number.pdf)
here are two files I generated. The author-year one doesn't actually create author year citations(???)

(FYI, on vacation through August 9; should have e-mail throughout, but responsiveness will vary)

**Comment by rmzelle** [comment]
@adam3smith, enjoy! Will hold the fort.

**Pull request closed without merging**